lib/sequin/accounts/accounts.ex

```diff
@@ -736,4 +736,66 @@ defmodule Sequin.Accounts do
 
     update_account(account, %{features: features})
   end
+
+  def invite_user(%User{} = user, %Account{} = account, sent_to, url_fun) when is_function(url_fun, 1) do
+    {encoded_token, user_token} = UserToken.build_account_invite_token(user, account.id, sent_to)
+
+    Ecto.Multi.new()
+    |> Ecto.Multi.run(:verify_account_ownership, fn repo, _ ->
+      query =
+        from au in AccountUser,
+          where: au.account_id == ^account.id and au.user_id == ^user.id
```

Comment:
Another pattern we like for common filters is to pass the query composition to the model module, like:

```suggestion
query =
account_id
|> AccountUser.where_account_id()
|> AccountUser.where_user_id(user_id)
```